client/me/account/main.jsx

```diff
@@ -529,6 +534,14 @@ const Account = React.createClass( {
 
 				{ this.communityTranslator() }
 
+				{ config.isEnabled( 'me/account/color-scheme-picker' ) &&
```

Comment:
I see that we're only enabling in development in this PR, so this is not a huge issue but to err on the side of caution what do you think to merging #17310 first and including the check in this PR? (See: https://github.com/Automattic/wp-calypso/pull/17310#pullrequestreview-59437674)

Reply:
Commenting for continuity: As per our discussion in Slack just now, we're going to leave the PR order as is to ensure we have a good test case for the feature detection in #17310. We're aiming to merge the PRs in quick succession.